Sources/SwiftDocC/Utility/Version.swift

```diff
@@ -35,7 +35,8 @@ public struct Version: Codable, RandomAccessCollection, ExpressibleByArrayLitera
     /// - warning: Returns `nil` if the version string contains non-integer or
     ///   negative numeric components; for example the strings "1.2.3-beta6" and "1.-2.3"
     ///   are invalid inputs.
-    public init?(versionString: String) {
+    public init?(versionString: String?) {
```

Comment:
Why is this change necessary? It's a breaking change.

Reply:
Back to the original init